app/model/testdata/CreateAdminData.scala

```diff
@@ -44,12 +45,17 @@ object CreateAdminData {
         val skills = createRequest.assessor.flatMap(_.skills).getOrElse(DataFaker.Random.skills)
         val sifterSchemes = createRequest.assessor.flatMap(_.sifterSchemes).getOrElse(DataFaker.Random.sifterSchemes)
         val civilServant = createRequest.assessor.flatMap(_.civilServant).getOrElse(DataFaker.Random.bool)
-        val availability: Option[List[AssessorAvailability]] = createRequest.assessor.flatMap(assessorRequest => {
+        val availability1: Option[List[AssessorAvailability]] = createRequest.assessor.flatMap(assessorRequest => {
             assessorRequest.availability.map { assessorAvailabilityRequests => {
               assessorAvailabilityRequests.map { assessorAvailabilityRequest => {
                 AssessorAvailability.apply(assessorAvailabilityRequest)
               }
-            }}}}).orElse(DataFaker.Random.Assessor.availability)
+            }}}})
+        val availability = createRequest.assessor match {
+          case Some(assessorRequest) if assessorRequest.status == AssessorStatus.AVAILABILITIES_SUBMITTED && availability1.isEmpty =>
```

Comment:
The way we handle the different statuses in the case of candidates is by creating another status generator class, i.e., so we have CreatedStatusGenerator, InProgressPersonalDetailsStatusGenerator, InProgressAssistanceDetailsStatusGenerator. 

The generation of admin and assessor is thought to work in the same way, although is not there yet, because we did not have statuses before.

Maybe we can have a discussion in person about this.

so in theory this logic should not be in CreateAdminData but